Content.Server/Audio/ContentAudioSystem.cs

```diff
@@ -27,8 +28,26 @@ public override void Initialize()
     {
         base.Initialize();
 
-        _lobbyMusicCollection = _prototypeManager.Index<SoundCollectionPrototype>(LobbyMusicCollection);
-        _lobbyPlaylist = ShuffleLobbyPlaylist();
+        //changes the music collection and reshuffles the playlist to update the lobby music
+        Subs.CVar(
+            _cfg,
+            CCVars.LobbyMusicCollection,
+            x =>
+            {
+                //Checks to see if the sound collection exists. If it does change it if not set a empty prototype
+                // as the new _lobbyMusicCollection meaning it wont play anything in the lobby.
+                if(_prototypeManager.TryIndex<SoundCollectionPrototype>(x, out var outputSoundCollection))
+                {
+                    _lobbyMusicCollection = outputSoundCollection;
+                }
+                else
+                {
+                    _lobbyMusicCollection = new SoundCollectionPrototype();
```

Comment:
Dynamically instantiating prototypes creates invalid objects and is not a good idea. Set the field to null if it's invalid.

Reply:
Also, this should report an error.